packages/workbox-core/INTERNAL/models/messages/messages.mjs

```diff
@@ -0,0 +1,19 @@
+export default {
+  'invalid-type': ({paramName, expectedType, value}) => {
```

Comment:
The bad scenario with this is that you end up with a message that doesn't make any sense.

> The parameter 'undefined' have an unexpected type. Expect type 'undefined'. Receive 'undefined'.

As an alternative, we can have a default message that is super generic (i.e. "A parameter had an unexpected type, please see stack trace for more info." then swap it out for "The paramter '${paramName}' ..." but the code quickly got verbose and just felt unusual).

I'm still not 100% happy with this way of generate useful error codes. Do you have any ideas of how this could be improved / what a better API would be?